test/disk_size_test.rb

```diff
@@ -277,4 +281,67 @@
     end
   end
 
+  describe "#ceil" do
+    it "returns the same value if any of the operands is zero" do
+      expect(zero.ceil(4.MiB)).to eq zero
+      expect(4.MiB.ceil(zero)).to eq 4.MiB
+      expect(zero.ceil(zero)).to eq zero
+    end
+
+    it "returns the same value if any of the operands is unlimited" do
+      expect(unlimited.ceil(8.GiB)).to eq unlimited
+      expect(8.GiB.ceil(unlimited)).to eq 8.GiB
+      expect(unlimited.ceil(zero)).to eq unlimited
+      expect(unlimited.ceil(unlimited)).to eq unlimited
+    end
+
+    it "returns the same value when rounding to 1 byte" do
+      expect(4.KiB.ceil(one_byte)).to eq 4.KiB
```

Comment:
This could accidentally succeed if you rounded to something else. Better test with 31337 bytes (which is prime)